README.md

```diff
@@ -5,6 +5,8 @@ s2i-nodejs-container 12 Quay.io status: [![Docker Repository on Quay](https://qu
 
 s2i-nodejs-container 14 Quay.io status: [![Docker Repository on Quay](https://quay.io/repository/centos7/nodejs-14-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/nodejs-14-centos7)
 
+s2i-nodejs-container 16 Quay.io status: [![Docker Repository on Quay](https://quay.io/repository/centos7/nodejs-16-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/nodejs-16-centos7)
+
```

Comment:
No centos7-based version for nodejs-16, so we will not be pushing any images in quay.io and this can be removed.

Reply:
thanks, this has been removed https://github.com/sclorg/s2i-nodejs-container/pull/295/commits/8b2d84a2da75798518f566c85a695a2574a34ce5